include/TrustWalletCore/TWTONSigner.h

```diff
@@ -0,0 +1,21 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+#pragma once
+
+#include "TWBase.h"
+//#include "TWTONProto.h"
+
+TW_EXTERN_C_BEGIN
+
+/// Helper class to sign TON transactions.
+TW_EXPORT_CLASS
+struct TWTONSigner;
+
+/// Signs a transaction.
+//TW_EXPORT_STATIC_METHOD
+//TW_TON_Proto_SigningOutput TWTONSignerSign(TW_TON_Proto_SigningInput input);
```

Comment:
let's enable this, account init msg is also kind of signing input, we can define different messages in .proto